src/imap_mag/plot/plot_ialirt_files.py

```diff
@@ -61,9 +61,31 @@ def create_figure(
     fig = plt.figure()
     gs = fig.add_gridspec(3, 4)
 
-    # HK dangers
+    # GSE science
     ax00 = fig.add_subplot(gs[0, 0:2])
 
+    for i, b in enumerate(
+        [
+            "mag_B_GSE_x",
+            "mag_B_GSE_y",
+            "mag_B_GSE_z",
+            "mag_B_magnitude",
+        ]
```

Comment:
Have you asked Tim what he wants plotted? For the I-ALiRT website they just plot Bz and |B| so worth checking, and all on one plot will be hard to view